modules/ROOT/pages/batch-error-handling-faq.adoc

```diff
@@ -1,23 +1,33 @@
-= Handling Errors During Batch Job
+= Handling Errors During Batch Jobs
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 == Overview
 
-Handling errors inside a batch job can be complicated considering the huge amount of data this scope is designed to take. +
-When processing 1 million records, it's almost impossible to log everything. Logs would become huge, unreadable, and would take a massive toll on the performance of your processing. +
-On the other hand, logging too little would make it impossible for you to know what went wrong. And considering that if out of those 1 million records, 30 thousand of them failed, not knowing what’s wrong with them can turn your debugging task very problematic. This trade-off is not simple to overcome.
+When a batch job processes very large chunks of data, verbose logs for issues that occur can become enormous and severely impact performance. 
 
+To limit this impact, Mule uses INFO-level log messaging by default, as described in <<default_logging>>. For cases in which you require more verbose log messages, you can change the mode to DEBUG. This mode is helpful not only for debugging but for some cases that involve smaller data sets. For example, you might use batch processing because it performs "near real-time" data integration, which recovers from crashes and continues processing a job from a point of failure. 
```

Comment:
```suggestion
To limit this impact, Mule uses INFO-level logging by default, as described in <<default_logging>>. For cases in which you require more verbose log messages, you can change the mode to DEBUG. This mode is helpful not only for debugging but for some cases that involve smaller data sets. For example, you might use batch processing because it performs "near real-time" data integration, which recovers from crashes and continues processing a job from a point of failure. 
```